docs/components/codeblock.md

```diff
@@ -96,6 +96,30 @@ Boolean to control whether to show or hide the search bar and related action but
 />
 ```
 
+### highlightedLineNumbers
+
+An array of line numbers to initially highlight in the code block. 
+
+If search is enabled, matching results will be highlighted in place of the provided line numbers, and all highlighted lines will be cleared once the search query is cleared.
```

Comment:
Why do we need to cancel line highlighting when the query is cleared? I think it's better to emit an event for this and let users to decide whether to cancel line highlighting.